packages/contracts-bedrock/src/cannon/MIPS64.sol

```diff
@@ -66,8 +66,8 @@ contract MIPS64 is ISemver {
     }
 
     /// @notice The semantic version of the MIPS64 contract.
-    /// @custom:semver 1.2.0
-    string public constant version = "1.2.0";
+    /// @custom:semver 1.2.1-beta.1
+    string public constant version = "1.2.1-beta.1";
```

Comment:
[Based on the versioning doc](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/book/src/policies/versioning.md#individual-contract-versioning), I don't think we're using the `-beta` suffixes anymore - I think this just goes to 1.3.0.